servicemesh-install/topics/install-supported-config.adoc

```diff
@@ -1,16 +1,16 @@
 [[supported-configurations]]
 = Supported configurations
 
-[IMPORTANT]
-====
-The current {ProductName} technology preview release is available for evaluation on Red Hat {product-title} version 3.11. For instructions, https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html[see the current installation documentation].
+// [IMPORTANT]
+// ====
+// The current {ProductName} technology preview release is available for evaluation on Red Hat {product-title} version 3.11. For instructions, https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html[see the current installation documentation].
 ====
```

Comment:
Please comment out this line.

Reply:
Got it!

Reply:
When did we remove the note that "OpenShift Online and OpenShift Dedicated are not supported for Red Hat OpenShift Service Mesh 0.#.TechPreview."?   Did that note get rewritten to what we just commented out here?  Because I think we still need to make it clear that Service Mesh is ONLY available on OpenShift Container Platform and NOT available on OpenShift Online or OpenShift Dedicated.

Reply:
@JStickler We didn't remove that information. That message is contained in the next two lines of documentation in the same file.

Reply:
@geekspertise D'oh!  Sorry, I somehow missed that.  Carry on then.  Don't mind me.